### PR TITLE
Set progress report interval for bazel explicitly

### PR DESCRIPTION
**Pull request by ericl**
It looks like sometimes we are hitting the 10m idle period, causing the build to exit early.

### CODE REVIEW

File: ci/keep_alive [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (ericl) [inline ci/keep_alive]: Seems like bazel sometimes will not print progress output even with the interval set, so use a script to make sure it happens.

**Comment by ericl** [comment]
@simon-mo this can be merged to reduce build flakiness

**Review by simon-mo — APPROVED** [review]
Looks good. Do you know why some of the error output are not reprinted at the end? I don't think there's a viable way to access the test log file like in https://travis-ci.com/ray-project/ray/jobs/259974065#L771-L773

**Comment by ericl** [comment]
I think it should be... Are you seeing some missing?

**Pull request merged**